CONTRIBUTING.rst

```diff
@@ -100,3 +73,51 @@ The full list of Ansible email lists and IRC channels can be found in the `commu
 .. _`freenode`: https://freenode.net
 .. _`molecule-users Forum`: https://groups.google.com/forum/#!forum/molecule-users
 .. _`communication page`: https://docs.ansible.com/ansible/latest/community/communication.html
+
+.. _community working group: https://github.com/ansible/community/tree/master/group-molecule
+
+Contribution Guidelines
+=======================
+
+* We are interested in various different kinds of improvement for Molecule;
+  please feel free to raise an `Issue`_ if you would like to work on something
+  major to ensure efficient collaboration and avoid duplicate effort.
+* Create a topic branch from where you want to base your work.
+* Check for unnecessary whitespace with ``git diff --check`` before committing.
```

Comment:
One can format changes as described in https://molecule.readthedocs.io/en/latest/testing.html?highlight=tox#formatting or check formatting with `tox -e format-check` which is what CI runs